src/main/kotlin/org/rust/cargo/runconfig/buildtool/CargoBuildTaskProvider.kt

```diff
@@ -26,6 +28,16 @@ class CargoBuildTaskProvider : RsBuildTaskProvider<CargoBuildTaskProvider.BuildT
     ): Boolean {
         if (configuration !is CargoCommandConfiguration) return false
         val buildConfiguration = getBuildConfiguration(configuration) ?: return true
+
+        val project = context.project ?: return false
```

Comment:
nit: 
```suggestion
        val project = configuration.project
```

or even inline it